### PR TITLE
Braav 8168/sample apple pay

### PR DESCRIPTION
**Pull request by antiv0**

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -626,6 +640,11 @@ export default class DefaultLayoutsClass extends Vue {
   showDrawer = false
 
   get title() {
+    if (this.isSafari && !this.appendApplePayOnce && (this.isStaging || this.issLocalHost)) {
+      // Apple Pay only works on safari
+      this.appendApplePayOnce = true;
+      this.paymentsLinks.push(this.applePayStaging)
```
Comment (kristinfritsch) [inline layouts/default.vue:646]:
```suggestion
      this.paymentsLinks = [...this.paymentsLinks, this.applePayStaging]
```

No need for `this.appendApplePayOnce`

File: layouts/default.vue [hunk 2]
```diff
@@ -626,6 +640,11 @@ export default class DefaultLayoutsClass extends Vue {
   showDrawer = false
 
   get title() {
+    if (this.isSafari && !this.appendApplePayOnce && (this.isStaging || this.issLocalHost)) {
+      // Apple Pay only works on safari
```
Comment (kristinfritsch) [inline layouts/default.vue:644]: This comment should go to the line above?

File: pages/debug/payments/applepay/create.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <v-btn
+        elevation="24"
+        dark
+        class="apple-pay-button apple-pay-button-text-pay"
+        v-on="on"
```
Comment (kristinfritsch) [inline pages/debug/payments/applepay/create.vue:6]: `on` is not defined, remove it?

**Review by kristinfritsch — COMMENTED** [review]
Just took a quick pass, haven't tested the functionality.
A few notes:
- please use the prettier extension to format the files
- add some documentation to the readme
- `lib/applePaySecrets.ts` and `lib/awsSSM.ts` should only be run on the server, move it to the `server` folder
-

File: layouts/default.vue [hunk 4]
```diff
@@ -203,9 +203,16 @@
...
 <script lang="ts">
 import { Vue, Component } from 'nuxt-property-decorator'
-
+import { getIsStaging, getIsLocalHost } from '@/lib/apiTarget'
+import { getIsSafari } from '@/lib/navigatorInfo'
 @Component
 export default class DefaultLayoutsClass extends Vue {
+  isStaging: boolean = getIsStaging()
+
+  issLocalHost: boolean = getIsLocalHost()
```
Comment (timmy-circle) [inline layouts/default.vue:212]: `isLocalHost` instead of `issLocalHost`?

**Comment by huaweigu** [comment]
all the secrets management will need to go to https://github.com/circlefin/payments-sample-app-private 
we don't want other people know how we manage secrets

**Comment by antiv0** [comment]
> Just took a quick pass, haven't tested the functionality. A few notes:
> 
> * please use the prettier extension to format the files
> * add some documentation to the readme
> * `lib/applePaySecrets.ts` and `lib/awsSSM.ts` should only be run on the server, move it to the `server` folder

**Pull request closed without merging**